pkg/apis/options/providers.go

```diff
@@ -6,6 +6,9 @@ const (
 
 	// OIDCGroupsClaim is the generic groups claim used by the OIDC provider.
 	OIDCGroupsClaim = "groups"
+
+	// OIDCRolesClaim is the generic roles claim used by the OIDC provider.
+	OIDCRolesClaim = "roles"
```

Comment:
Overall the code quality is good. It just seems questionable to me to add this claim in such a prominent and generic place as it is not defined in the OIDC spec and not as widely used as the groups claim.